### ISSUE TITLE
If run at the same time as one slot effect, it will crash

### ISSUE THREAD

**Issue by huzake** [xNaxdy/latency-slider-de#18]
Crash when closing arena room.Could you please update it to be compatible with one slot effect?

**Comment by xNaxdy** [comment]
Crashing when *closing* an arena room I've heard a lot from people who forgot to remove the old latency slider mod. Can you double check to ensure `libzlatency_slider_de.nro` is the only latency slider type mod you have?

**Comment by huzake** [comment]
Yes, I checked various things using ultimate plugin nro.This result only occurs when latency_slider_de and one slot effect are executed at the same time. The same is true when playing VIP matches.However, we have not yet confirmed v0.2.3.

**Comment by xNaxdy** [comment]
Noted. Let me know if it still occurs with v0.2.3.

**Comment by huzake** [comment]
When I checked, it crashed as well.

**Comment by huzake** [comment]
After further investigation, I found the cause. It seems that I inserted an old one slot effect when I reinstalled skyline.Quick notes since I keep getting asked this: I noticed this too late because I was having similar symptoms.Thank you for responding to my question.I'll be careful next time
